rust/operator-binary/src/controller.rs

```diff
@@ -919,6 +939,21 @@ fn env_var_from_discovery_config_map(
     })
 }
 
+fn env_var_from_secret(secret_name: &Option<String>, env_var: &str) -> Option<EnvVar> {
```

Comment:
Would be something for operator-rs I guess ;)
You can add it there if you want to but I'm fine with it here as we

Reply:
yeah but i dont wanna release a new operator-rs  for now.... :D